modules/deployment-scripts/create-kv-certificate/main.bicep

```diff
@@ -39,9 +39,26 @@ param initialScriptDelay string = '0'
 @description('When the script resource is cleaned up')
 param cleanupPreference string = 'OnSuccess'
 
-var neededRBACRoles = !empty(rbacRolesNeededOnKV) ? rbacRolesNeededOnKV : 'a4417e6f-fecd-4de8-b567-7b0420556985'
+@description('Self, or {IssuerName} for certificate signing')
+param issuerName string = 'Self'
 
-resource akv 'Microsoft.KeyVault/vaults@2021-11-01-preview' existing = {
+@description('Certificate Issuer Provider, DigiCert, GlobalSign, or internal options may be used.')
+param issuerProvider string = ''
+
+@description('Account ID of Certificate Issuer Account')
+param accountId string = ''
+
+@description('Password of Certificate Issuer Account')
+@secure()
+param digicertPassword string = ''
```

Comment:
probably need to genericise the param name further as it can be used for issuers other than just digicert